### PR TITLE
Improve code coverage for array and trilu

### PR DESCRIPTION
**Pull request by yimoj**
- Improve code coverage
- Change built-in name `object` to `obj`

### CODE REVIEW

File: tests/integration/test_trilu_indices.py [hunk 1]
```diff
@@ -192,23 +192,23 @@ class TestTriluIndicesFromErrors:
...
         a = num.ones(shape, dtype=int)
-        msg = "input array must be 2-d"
-        with pytest.raises(ValueError, match=msg):
-            num.tril_indices_from(a)
-
-    def test_arr_0d(self):
-        a = num.array(3)
-        msg = "input array must be 2-d"
-        with pytest.raises(ValueError, match=msg):
-            num.tril_indices_from(a)
```
Comment (bryevdv) [inline tests/integration/test_trilu_indices.py:206]: What's the reason for deleting this test? (This case doesn't seem to be covered anywhere else)

Comment (yimoj) [reply tests/integration/test_trilu_indices.py:206]: It was an unfortunate accident, adding it back with the intended changes.

**Comment by bryevdv** [comment]
@yimoj any ideas about the test failures? Maybe merge in the latest to kick off CI fresh?

**Comment by yimoj** [comment]
> @yimoj any ideas about the test failures? Maybe merge in the latest to kick off CI fresh?

Probably because the base changed, I've rebased to 23.07 to kick off another run.

**Comment by yimoj** [comment]
@bryevdv looks like the culprit is from this commit on CI:
https://github.com/nv-legate/cunumeric/commit/01836a0774a468f6bc0066e7bed799edddd5328d

**Comment by yimoj** [comment]
https://github.com/nv-legate/cunumeric/pull/943 CI should be fine now, I'll trigger the tests again.

**Review by bryevdv — APPROVED** [review]

**Pull request merged**